client/a8c-for-agencies/sections/marketplace/checkout/style-v1.scss

```diff
@@ -29,6 +29,10 @@
 			var( --color-neutral-0 ) 100%
 		);
 	}
+
+	.hosting-dashboard-layout__body-notices {
```

Comment:
Since the padding top for the body on Checkout is set to `0`, we are setting the padding top for the banners